libtsuba/src/RDGPartHeader.h

```diff
@@ -432,9 +432,9 @@ class KATANA_EXPORT RDGPartHeader {
     for (const auto& name : names) {
       auto it = name_to_slot.find(name);
       if (it == name_to_slot.end()) {
-        return KATANA_ERROR(
-            ErrorCode::PropertyNotFound, "no property named {}",
-            std::quoted(name));
+        KATANA_LOG_DEBUG(
+            "Non-existant property \"{}\" requested. Skipping.", name);
+        continue;
```

Comment:
Sweet; Probably a matter of discussion but I would prefer `KATANA_LOG_WARN` here. I think this message would be helpful in customer engagements where we are trying to diagnose something and we pretty much never distribute DEBUG builds.

Reply:
Makes sense to me. I'll change it.